### ISSUE TITLE
KVTable, etc

### ISSUE THREAD

**Issue by davalapar** [davalapar/ruruDB#5]
### KVTable
- Just like maps
- [x] set
- [x] get
- [x] has
- [x] delete

### Table
- [x] Remove `mustExist`, checks must be done upon `await db.initialize()`

### Converters
- [x] Saved tables must be versioned so we can easily convert old formats to new formats during loading phase

### Database
- [x] Snapshots must be sourced from `main` file, not `old` file
- [x] Why are there defaults at https://github.com/davalapar/ruruDB/blob/master/src/main.ts#L390

**Comment by davalapar** [comment]
Save format:

```
[
  [  filename, rurudbVersion ],
  [  [label, items], n ], // tables
  [  [label, keys, values], n ] // kvtables
]
```

**Comment by davalapar** [comment]
Closed @ 7.0.0
